sos/plugins/s390.py

```diff
@@ -27,7 +27,7 @@ class S390(Plugin, RedHatPlugin):
     # Check for s390 arch goes here
 
     def check_enabled(self):
-        return (self.policy().get_arch() == "s390")
+        return (self.policy().get_arch() in ["s390", "s390x"])
```

Comment:
I'll amend this to say:

```python
    return ("s390" in self.policy().get_arch())
```

When I merge - after thinking about it some I think this looks cleaner / simpler and there is no real risk of collisions.